examples/view-integration/view-framework-sampler/webpack.config.cjs

```diff
@@ -47,15 +47,21 @@ module.exports = (env) => {
 				new HtmlWebpackPlugin({
 					template: "./src/index.html",
 				}),
-				// new CleanWebpackPlugin(),
+				new webpack.DefinePlugin({
+					// These are not required, but the Vue docs recommend setting them.
+					// See https://vuejs.org/api/compile-time-flags.html#webpack
+					__VUE_OPTIONS_API__: "true",
+					__VUE_PROD_DEVTOOLS__: "false",
+					__VUE_PROD_HYDRATION_MISMATCH_DETAILS__: "false",
+				}),
 			],
 			resolve: {
 				extensionAlias: {
 					".js": [".ts", ".tsx", ".js", ".cjs", ".mjs"],
 				},
 				extensions: [".ts", ".tsx", ".js", ".cjs", ".mjs"],
 				alias: {
-					vue$: "vue/dist/vue.esm.js",
+					vue$: "vue/dist/vue.esm-bundler.js",
```

Comment:
What is this exactly? I see it has `dist` in the path. And `vue` before `dist`.
This also appears to be the second `resolve` entry in this object. That is allowed and the first is ignored?

Reply:
Not a Vue expert (esp. v3) but this is the Vue file to use with Bundlers according to https://github.com/vuejs/core/tree/main/packages/vue#with-a-bundler. Without this, the template we define in the Vue component cannot be rendered at runtime. We could switch to a compile time template I think but it's more churn.

Related: https://vuejs.org/api/options-rendering.html#template

> A template provided via the template option will be compiled on-the-fly at runtime. It is only supported when using a build of Vue that includes the template compiler. The template compiler is NOT included in Vue builds that have the word runtime in their names, e.g. vue.runtime.esm-bundler.js.

I didn't notice the duplicated resolve - fixed in latest.